charts/linkerd2/templates/prometheus.yaml

```diff
@@ -50,6 +50,13 @@ data:
 
       kubernetes_sd_configs:
       - role: node
+      metric_relabel_configs:
```

Comment:
since `metric_relabel_configs` happen after `relabel_configs` are applied, mind moving this below that section?